README.md

```diff
@@ -97,5 +112,7 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
 ```
 
+Mo
+
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please try playing with for loops the way Carlos did in class.